packages/valory/skills/dynamic_nft_abci/handlers.py

```diff
@@ -290,7 +291,7 @@ def _handle_get_health(
 
             seconds_since_last_reset = current_time - last_update_time
             seconds_until_next_update = (
-                observation_interval - seconds_since_last_reset
+                AVERAGE_PERIOD_SECONDS + observation_interval - seconds_since_last_reset
```

Comment:
We were not taking into account that observation_interval is the wait time (imo we should rename it). So to estimate when the next update will happen, we need both the average period time + the wait time.